Odata-docs/extensions/migration.md

```diff
@@ -0,0 +1,262 @@
+﻿# Introduction
+
```

Comment:
the document is also missing meta information that helps discover-ability. please resolve these two items:

[Warning] Missing attribute: author. Add the current author's GitHub ID.
[Warning] Missing attribute: title. Add a title string to show in search engine results.

Reply:
added